doc/oemof_solph.rst

```diff
@@ -432,11 +432,53 @@ Furthermore, an efficiency for loading, unloading and a capacity loss per time i
         capacity_loss=0.001, nominal_capacity=50,
         inflow_conversion_factor=0.98, outflow_conversion_factor=0.8)
 
+
+Using the investment object with the GenericStorage component
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+The `GenericInvestmentStorageBlock` enables the investment mode for storages to optimise dispatch and investment. It is 
+based on the `GenericStorage` component and there are two main investment possibilities.
+
+    *	Invest into the flow parameters e.g. a turbine or a pump
+    *	Invest into capacity of the storage  e.g. a basin or a battery cell
+    
+As an addition to other flow-investments, the storage class implements the possibility to couple or decouple the flows 
+with the capacity of the storage. 
+Three parameters are responsible for connecting the flows and the capacity of the storage:
+
+    *	' `invest_relation_input_capacity` ' fixes the inflow to the capacity. A ratio of ‘1’ means that the storage can be emptied within one time-period. 
+    *	' `invest_relation_output_capacity` ' fixes the outflow to the capacity. A ratio of ‘1’ means that the storage can be emptied within one period.
+    *	' `invest_relation_input_output` ' fixes the input flow to the output flow. For values <1, the input will be smaller and for values >1 the input flow will be larger. 
+    
+Not all 3 parameters can be set at the same time. 
```

Comment:
I think there is an error or a warning if all three are set but I would rather explain that is overdetermined and therefore error prone if you use all three parameter.